### ISSUE TITLE
Visible mesh structure

### ISSUE THREAD

**Issue by runa91** [facebookresearch/pytorch3d#106]
When rendering a mesh the resulting image shows highlighted edges as in the attached picture. How can this be fixed?
![img_001530](https://user-images.githubusercontent.com/15121854/76204631-0f51ee00-61f9-11ea-92b4-7efcf19d80cb.png)

**Comment by nikhilaravi** [comment]
@runa91 what shader/blending function are you using? The images output from the shaders are RGBA, and that edge pattern is probably due to how the alpha channel is blended in the softmax blending function - to fix the image above, you can try viewing only the RGB component i.e. `images[..., :3]`.

**Comment by runa91** [comment]
It was indeed a visualization issue. Thanks for your response.

**Comment by Bob-Yeah** [comment]
It is the same case when rendering silhouete image(using default SoftSilhouetteShader) especially with _cull_backfaces=True_ and low _faces_per_pixel._ 
Any advice how I can remove these artifacts?
faces_per_pixel=100
![facesperpixel100](https://user-images.githubusercontent.com/12245304/81467997-20c27100-920f-11ea-9340-bc1699cb4d52.png)
faces_per_pixel=10
![facesperpixel10](https://user-images.githubusercontent.com/12245304/81467996-2029da80-920f-11ea-8c09-ea15cf997f2a.png)
cull_backfaces=True
![withcullbackface](https://user-images.githubusercontent.com/12245304/81467998-20c27100-920f-11ea-9a08-3952b7748ccb.png)

**Comment by pableeto** [comment]
@Bob-Yeah I have similar issue here. Have you managed to solve it? Thanks.
